pkg/models/target_write_result_test.go

```diff
@@ -40,30 +40,10 @@ func TestNewTargetWriteResult_EmptyWithoutTime(t *testing.T) {
 	assert.Equal(time.Duration(0), r.MaxTransformLatency)
 	assert.Equal(time.Duration(0), r.MinTransformLatency)
 	assert.Equal(time.Duration(0), r.AvgTransformLatency)
-}
-
-// TestNewTargetWriteResult_EmptyWithTime tests that an empty targetWriteResult with no a provided timestamp will report 0s across the board
-func TestNewTargetWriteResult_EmptyWithTime(t *testing.T) {
```

Comment:
Has this test case been removed entirely?

Reply:
Yes. I deleted it because it looks like it's testing same thing as the one above (`TestNewTargetWriteResult_EmptyWithoutTime`)?

You know, I like 'more tests' but not like that!! 😁

Reply:
Yeah they're identical, not sure what I was thinking here. Probably just an oversight - I'm happy to remove, thank you!